docs/install_guides/pypi-openvino-dev.md

```diff
@@ -78,20 +84,32 @@ python -m pip install --upgrade pip
 
 ### Step 4. Install the Package
 
-Run the command below: <br>
+To install and configure the components of the development package for working with specific frameworks, use the `pip install openvino-dev[extras]` command, where `extras` is a list of extras from the table below: 
+
+| DL Framework                                                                     | Extra                          |
+| :------------------------------------------------------------------------------- | :-------------------------------|
+|   [Caffe*](https://caffe.berkeleyvision.org/)                                    | caffe                           |
+|   [Caffe* 2](https://caffe2.ai/)                                                 | caffe2                          |
+|   [Kaldi*](https://kaldi-asr.org/)                                               | kaldi                           |
+|   [MXNet*](https://mxnet.apache.org/)                                            | mxnet                           |
+|   [ONNX*](https://github.com/microsoft/onnxruntime/)                             | onnx                            |
+|   [PyTorch*](https://pytorch.org/)                                               | pytorch                         |
+|   [TensorFlow* 1.x](https://www.tensorflow.org/versions#tensorflow_1)            | tensorflow                      |
+|   [TensorFlow* 2.x](https://www.tensorflow.org/versions#tensorflow_2)            | tensorflow2                     |
 
+For example, to install and configure the components for working with TensorFlow 2.x, MXNet and Caffe, use the following command:  
    ```sh
-   pip install openvino-dev
+   pip install openvino-dev[tensorflow2,mxnet,caffe]
    ```
 
-### Step 5. Verify that the Package is Installed
+### Step 5. Verify that the Package Is Installed
 
 Run the command below (this may take a few seconds):
 ```sh
-pot -h
+mo -h
```

Comment:
When I installed openvino-dev 2021.4 on Ubuntu 20.4 with Python 3.7 `mo -h` worked: it showed the help file as expected. But `from openvino.inference_engine import IECore` failed, because I had not installed libpython3.7-dev. Is there another command that can be used here that would catch that error?

Reply:
Please find an answer in Troubleshooting section of this document.

Reply:
Sadly that doesn't work on Ubuntu 20.  It is not trivial to install Python 3.7 libraries there. Much easier to just use Python 3.8. But also, it is nice if the verification command verifies that the installation indeed worked, and OpenVINO works as expected.

Reply:
1. Python default version on Ubuntu 18.04 and Ubuntu 20.04
Python 3.6 is default version on [Ubuntu 18.04](https://packages.ubuntu.com/bionic/allpackages) with ability to install  Python 3.7 from Ubuntu repository
Python 3.8 is default version on [Ubuntu 20.04](https://packages.ubuntu.com/focal/allpackages) and Python 3.7 is absent.
So I clarified the Ubuntu version in troubleshooting instructions.
2. Added additional verification step for mentioned issue (the same as defined for runtime package).